lib/engine/round/operating.rb

```diff
@@ -74,6 +74,8 @@ def recalculate_order
         # to change order. Re-sort only them.
         index = @entity_index + 1
         @entities[index..-1] = @entities[index..-1].sort if index < @entities.size - 1
+
+        @entities.pop while @entities.last.corporation? && @entities.last.share_price.liquidation?
```

Comment:
would this break if entities is empty?

Reply:
It does... I don't know how I missed that :disappointed: